docs/intents.rst

```diff
@@ -184,7 +184,7 @@ For example:
     # from nextcord.ext import commands
     # bot = commands.Bot(command_prefix='!', intents=intents)
 
-What happened to my message commands?
+What happened to my text commands?
```

Comment:
ext.commands-style commands are typically called either "message commands" or "prefix commands"

```suggestion
What happened to my prefix commands?
```

Reply:
I will say `prefix commands` since it could cause conflict with the Message Commands that are a part of application commands.